src/sentry/deletions/defaults/group.py

```diff
@@ -25,34 +26,33 @@ def chunk(self):
         if self.last_event is not None:
             conditions.extend(
                 [
-                    ["timestamp", "<=", self.last_event.timestamp],
+                    ["timestamp", "<=", self.last_event["timestamp"]],
                     [
-                        ["timestamp", "<", self.last_event.timestamp],
-                        ["event_id", "<", self.last_event.event_id],
+                        ["timestamp", "<", self.last_event["timestamp"]],
+                        ["event_id", "<", self.last_event["event_id"]],
                     ],
                 ]
             )
 
-        events = eventstore.get_events(
-            filter=eventstore.Filter(
-                conditions=conditions, project_ids=[self.project_id], group_ids=[self.group_id]
-            ),
-            limit=self.DEFAULT_CHUNK_SIZE,
+        events = raw_query(
+            selected_columns=["event_id", "timestamp"],
+            filter_keys={"project_id": [self.project_id], "group_id": [self.group_id]},
+            conditions=conditions,
             referrer="deletions.group",
             orderby=["-timestamp", "-event_id"],
-        )
+        )["data"]
```

Comment:
I don't think we should bypass eventstore to fetch events. See my comment on PR https://github.com/getsentry/sentry/pull/16284.

Let's first see if loading everything from eventstore is a problem. Maybe we are fine with a multi_get. If it is a problem we should revisit the issue of loading a small number of fields on a big number of events through eventstore.

Reply:
I'll revert this commit, and we can observe if it proves to be a problem before attempting something like this.